var/spack/repos/builtin/packages/emboss/package.py

```diff
@@ -16,9 +16,19 @@ class Emboss(AutotoolsPackage):
 
     version('6.6.0', 'cc3fca80cb0618deb10fa0d29fe90e4b')
 
-    depends_on('libxpm')
+    variant('without-x', default=False, description='Build without X support')
```

Comment:
I would call this X or x11 (there's evidence for both in existing packages:

```
ag 'variant\(.*with' var
```
and the other `postgresql` or `postgres`

```
ag "variant\('postg" var
```